### DISCUSSION TITLE
how to set web search?

### DISCUSSION THREAD

**Discussion by wayen84** [reworkd/AgentGPT#1268] [General]
<img width="566" alt="截屏2023-09-06 12 04 20" src="https://github.com/reworkd/AgentGPT/assets/39221727/57fcb607-3d53-4f72-967f-02f08010ea68">

There is no any selection for web search.

**Comment by jasangill1** [comment]
Hello @wayen84 do you have the paid version ?

**Reply by wayen84** [reply]
Nope. You mean that only the paid version is with web search?
